src/core_plugins/kibana/public/discover/__tests__/directives/field_chooser.js

```diff
@@ -104,21 +104,19 @@ describe('discover field chooser directives', function () {
   };
 
   describe('Index list', function () {
-    it('should be in alphabetical order', function (done) {
```

Comment:
__note:__ removing redundant `done` from few tests is the only side thing that's included in this PR.